packages/fx-core/src/component/driver/aad/interface/createAadAppArgs.ts

```diff
@@ -4,4 +4,5 @@
 export interface CreateAadAppArgs {
   name: string; // The name of AAD app
   generateClientSecret: boolean; // Whether generate client secret for the app
+  signInAudience?: string; // Specifies what Microsoft accounts are supported for the current application.
```

Comment:
is it possible to mark signInAudience to be `SignInAudience` type?

Reply:
Updated, thx!